prow/config.yaml

```diff
@@ -153,3 +153,51 @@ presubmits:
       - name: cache-ssd
         hostPath:
           path: /mnt/disks/ssd0
+
+postsubmits:
+  istio/pilot:
+  - name: pilot-postsubmit
+    branches:
+    - master
+    spec:
+      containers:
+      - image: gcr.io/istio-testing/prowbazel:0.1.5
+        args:
+        - "--branch=$(PULL_REFS)"
+        - "--upload=gs://kubernetes-jenkins/logs"
+        - "--git-cache=/root/.cache/git"
```

Comment:
it caches the git repo so you may not need to pull all the objects again, but I mistakenly included the flag.